js/screens/authentication/IdpSelectionScreen.js

```diff
@@ -101,7 +102,10 @@ class IdpSelectionScreen extends React.Component<Props> {
   render(): React.Node {
     return (
       <Container>
-        <Header iosBarStyle="dark-content">
+        <Header
```

Comment:
since this customized `Header` is repeated in 3 places, I would make a custom component

Reply:
I am trying to make this app wide using only theme/variables. The problem i have is that getter methods can't be overridden using a simple object merge, and the iosBarStyle is using, in native-base theme, a getter method to get the proper value.

Reply:
I understand, what I meant was just a simple

```js
export function AppHeader(props) {
 return <Header iosBarStyle="dark-content" ... >{props.children}</Header>
}
```